### ISSUE TITLE
No Hardware Acceleration after compiling from source 

### ISSUE THREAD

**Issue by BuksheeTester** [moonlight-stream/moonlight-qt#114]
So I had a struggle getting the app to work using Flatpak no matter what I do I keep getting the errors

libGL error: No matching fbConfigs or visuals found
libGL error: failed to load driver: swrast
00:00:00 - Qt Fatal: Could not initialize GLX

I have tried installing the appropriate Nvidia packages and various other troubleshooting options which are now too numerous to remember right now but still no joy.

Very much the same issues mentioned in issue #49 

Decided to have a go at compiling using QT creator and after a steep learning curve (I am not an expert in any measurable sense) I now have a (mostly) working application. I even managed to create a working shortcut to run it straight from the desktop (go me!)

However, I am getting the error notification about the lack of hardware acceleration after starting the app.

I was wondering if there is a step I may have missed or something I can do to correct this. All I have done is to install QT and the other packages as mentioned in the build instructions and then build moonlight. The error is present after starting the app from the app/ sub-folder. 

OS Ubuntu 18.04

Moonlight settings are completely default.

For now I have a working application so am happy and will continue to watch for any more posts in the relevant issues here regarding flatpak but if anyone has an idea I can try then I will be grateful.

**Comment by cgutman** [comment]
can you post your moonlight log? It will show up in the terminal window if you run moonlight from the terminal.

**Comment by cgutman** [comment]
My guess is that VDPAU support wasn't compiled into FFmpeg or Moonlight. Did you have `libvdpau-dev` installed when you built? Where did you get your FFmpeg build?

**Comment by BuksheeTester** [comment]
You are indeed correct!

Ok so I confirmed that I have libvdpau-dev installed
I previously downloaded FFmpeg as per the build instructions (http://ffmpeg.org/)

So my next step was to recompile FFmpeg and then I rebuilt the app again from scratch using QT Creator.

I can now start the moonlight without any notice about Hardware Acceleration issues. Perfect!

My guess is that my order of installations got mixed up. Thanks for nudging me in the right direction, very much appreciated!
